pkg/client/cmd/app.go

```diff
@@ -56,6 +56,9 @@ func createApp(cmd *cobra.Command, args []string) {
 		return
 	}
 	name := args[0]
+	if len(name) > 63 {
```

Comment:
Try to avoid magic numbers, at least put in a constant with a descriptive name is a good idea.